modules/bemenu/hm.nix

```diff
@@ -1,30 +1,27 @@
-{
-  mkTarget,
-  lib,
-  config,
-  ...
-}:
+{ mkTarget, lib, ... }:
 mkTarget {
   name = "bemenu";
   humanName = "bemenu";
 
-  extraOptions = {
-    fontSize = lib.mkOption {
-      description = ''
-        Font size used for bemenu.
-      '';
-      type = with lib.types; nullOr int;
-      default = config.stylix.fonts.sizes.popups;
-    }; # optional argument
+  extraOptions =
+    { fonts }:
+    {
+      fontSize = lib.mkOption {
+        description = ''
+          Font size used for bemenu.
+        '';
+        type = with lib.types; nullOr int;
+        default = fonts.sizes.popups;
```

Comment:
Not part of this PR, but if you're fixing issues in this file this default also needs a defaultText.